ubuntu/14.04/standard/Dockerfile

```diff
@@ -62,12 +71,16 @@ RUN apt-get update \
 RUN apt-get update \
  && apt-get install -y --no-install-recommends openjdk-8-jdk \
  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends openjdk-9-jdk \
+ && rm -rf /var/lib/apt/lists/*
 RUN apt-get update \
  && apt-get -o Dpkg::Options::="--force-overwrite" install -y --no-install-recommends openjdk-8-jdk \
  && rm -rf /var/lib/apt/lists/*
 RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
 ENV JAVA_HOME_7_X64=/usr/lib/jvm/java-7-openjdk-amd64 \
     JAVA_HOME_8_X64=/usr/lib/jvm/java-8-openjdk-amd64 \
+    JAVA_HOME_9_X64=/usr/lib/jvm/java-9-openjdk-amd64 \
     JAVA_HOME_8_X64=/usr/lib/jvm/java-8-openjdk-amd64 \
```

Comment:
Not new in your PR, but why is JAVA_HOME_8_X64 set twice?

Reply:
I saw that after I created my pull request. I'll remove it with my next commit.

Reply:
I believe it is set multiple times intentionally by the update.sh script.  It's compensating for update.sh handling different versions of Java on Ubuntu 14.04 and 16.04.